_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -101026,6 +101061,13 @@
 	},
 /turf/open/floor/iron,
 /area/security/office)
+"uVT" = (
+/obj/item/clothing/under/color/red,
+/obj/effect/decal/remains/human,
+/turf/open/floor/plating,
+/area/maintenance/port/fore{
+	desc = "How did...   What the hell?"
+	})
```

Comment:
```suggestion
/area/maintenance/port/fore)
```

Reply:
can't edit vars with /areas

Reply:
I remember removing that desc, weird that it saved that. Looking into it.